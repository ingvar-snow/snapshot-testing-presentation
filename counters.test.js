


import getCounters from './counters.js';

test('getting counters for mobile platform', () => {

  expect(getCounters('mobile')).toMatchSnapshot();
  
});



