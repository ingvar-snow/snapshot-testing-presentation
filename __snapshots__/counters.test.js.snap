// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getting counters for mobile platform 1`] = `
Array [
  Object {
    "id": "22222222",
    "platform": "mobile",
    "type": "yandex-metrika",
    "webvisor": true,
  },
  Object {
    "id": "33333333",
    "platform": "mobile",
    "type": "yandex-metrika",
    "webvisor": false,
  },
]
`;
